libsrc/c64/ser/c64-up2400.s

```diff
@@ -0,0 +1,479 @@
+; Serial driver for the C64 using Userport.
+;
+; Johan Van den Brande, (c) 2014
+; 
+; Based on George Hug's 'Towards 2400' article in the Transactor Magazine volume 9 issue 3.
+; (https://archive.org/details/transactor-magazines-v9-i03) 
+;
+        .include        "cbm_kernal.inc"
+        .include        "zeropage.inc"
+        .include        "ser-kernel.inc"
+        .include        "ser-error.inc"
+        .include        "c64.inc"
+
+        .macpack        module
+
+; ------------------------------------------------------------------------
+; Constants
+RecvHead = $b5          ; Head of receive buffer
```

Comment:
Thanks for starting to add (more) constants. It would be nice if the names would adhere to the same style. Either CamelCase for all or for none (`RecvHead` vs. `ribuf`).

Reply:
I was using the label of c64-swlink.s...